.drone.star

```diff
@@ -26,192 +22,161 @@ def testing(ctx):
-            },
-            {
-                'name': 'dependencies',
-                'image': 'owncloudci/nodejs:14',
-                'commands': [
-                    'yarn install'
-                ],
-                'depends_on': ['clone']
-            },
-            {
-                'name': 'eslint',
-                'image': 'owncloudci/nodejs:14',
-                'commands': [
-                    'yarn lint:eslint',
-                ],
-                'depends_on': ['dependencies']
-            },
-            {
-                'name': 'stylelint',
-                'image': 'owncloudci/nodejs:14',
-                'commands': [
-                    'yarn lint:stylelint',
-                ],
-                'depends_on': ['dependencies']
-            },
-            {
-                'name': 'unit tests',
-                'image': 'owncloudci/nodejs:14',
+        {
+            'name': 'eslint',
+            'image': 'owncloudci/nodejs:14',
+            'commands': [
+                'yarn lint:eslint',
+            ],
+            'depends_on': ['dependencies']
+        },
+        {
+            'name': 'stylelint',
+            'image': 'owncloudci/nodejs:14',
+            'commands': [
+                'yarn lint:stylelint',
+            ],
+            'depends_on': ['dependencies']
+        },
+        {
+            'name': 'unit tests',
+            'image': 'owncloudci/nodejs:14',
+            'commands': [
+                'yarn run tokens',
+                'yarn test',
+            ],
+            'depends_on': ['eslint', 'stylelint']
+        },
+        {
+            'name': 'sonarcloud',
+            'image': 'sonarsource/sonar-scanner-cli:latest',
+            'environment': sonar_env,
+            'depends_on': ['unit tests']
+        },
+        {
+            'name': 'build docs',
+            'image': 'owncloudci/nodejs:14',
                 'commands': [
-                    'yarn run tokens',
-                    'yarn test',
-                ],
-                'depends_on': ['eslint', 'stylelint']
-            },
-            {
-              'name': 'sonarcloud',
-              'image': 'sonarsource/sonar-scanner-cli:latest',
-              'pull': 'always',
-              'environment': sonar_env,
-              'depends_on': ['unit tests']
-            },
-            {
-                'name': 'build docs',
-                'image': 'owncloudci/nodejs:14',
-                'pull': 'always',
+                'yarn build:docs',
+            ],
+            'depends_on': ['unit tests']
+        },
+        {
+            'name': 'build system',
+            'image': 'owncloudci/nodejs:14',
                 'commands': [
-                    'yarn build:docs',
-                ],
-                'depends_on': ['unit tests', 'eslint', 'stylelint']
+                'yarn build:system',
+            ],
+            'depends_on': ['unit tests']
+        },
+    ]
+
+    release_steps = [
```

Comment:
Release steps are only used on tag. IMO better to move construction of this array into the `if` statement body starting in line 153 (set the name first, then add all these additional release steps to the final_steps array).